index.html

```diff
@@ -29,12 +35,133 @@ <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
+                </div>
             </div>
+            <div class="project pr-3">
+                <div class="snap-shoot">
+                    <img src="./image/Snapshoot Portfolio-3.svg" alt="project 1">
+                </div> 
+                <div class="info">
+                    <h2 class="pj-title pt-1">Multi-Post Stories</h2>
+                    <ul class="meta">
+                        <li class="type">Facebook</li>
+                        <li class="catagories gray"><span class="dot"></span>Full Stack Dev</li>
+                        <li class="year gray"><span class="dot"></span>2015</li>
+                    </ul>
+                    <p class="project-des">
+                        Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+                    </p>
+                    <ul class="langue">
+                        <li>HTML</li>
+                        <li>Ruby on rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="read-more"><a href="http://"> See Project</a></button>
+                </div>
+            </div>
+            <div class="project pr-4">
+                <div class="snap-shoot">
+                    <img src="./image/Snapshoot Portfolio-4.svg" alt="project 1">
+                </div> 
+                <div class="info">
+                    <h2 class="pj-title pt-1">Facebook 360</h2>
+                    <ul class="meta">
+                        <li class="type">Facebook</li>
+                        <li class="catagories gray"><span class="dot"></span>Full Stack Dev</li>
+                        <li class="year gray"><span class="dot"></span>2015</li>
+                    </ul>
+                    <p class="project-des">
+                        Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                    </p>
+                    <ul class="langue">
+                        <li>HTML</li>
+                        <li>Ruby on rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="read-more btn"><a href="http://"> See Project</a></button>
+                </div>
+            </div>
+        </section>
+        <section class="about">
+            <div class="about-info">
+                <h2>About <br> Myself</h2>  
+                <p class="about-des">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>  
+                <div class="social">
+                    <p>let's connect</p>
+                    <ul class="social-icon">
```

Comment:
- [ ] Kindly change the paragraph tag used to wrap `let's connect` to header tags from (h3 - h6) to show your design differs from the paragraph :+1: